src/__tests__/keywordIntegration.test.ts

```diff
@@ -0,0 +1,106 @@
+/* eslint-disable no-console */
+import { gql } from 'apollo-server';
+
+import KeywordAPI from '../datasources/keyword';
+import { getApolloTestServer } from '../utils/testUtils';
+
+const GET_KEYWORDS = gql`
+  query KewordList($source: LinkedEventsSource) {
+    keywordList(source: $source) {
+      data {
+        id
+        internalId
+        name {
+          fi
+        }
+      }
+      meta {
+        count
+      }
+    }
+  }
+`;
+
+it('resolves keywordList correctly', async () => {
+  const keywordAPI = new KeywordAPI();
+
+  const keywordResponse = {
+    data: [
+      {
+        id: '1',
+        '@id': '1',
+        name: {
+          fi: 'keyword1',
+        },
+      },
+      {
+        id: '2',
+        '@id': '2',
+        name: {
+          fi: 'keyword2',
+        },
+      },
+    ],
+    meta: {
+      count: 1,
+    },
+  };
+
+  (keywordAPI as any).get = jest.fn().mockResolvedValue(keywordResponse);
+
+  const { query } = getApolloTestServer({
+    dataSources: () => ({ keywordAPI }),
+  });
+
+  const res = await query({ query: GET_KEYWORDS });
+
+  if (res.errors) console.log(res.errors);
+
+  expect(res.data.keywordList).toEqual({
```

Comment:
You can also have `expect(res.data.keywordList).toEqual(keywordResponse)` or `expect(res).toMatchSnapshot();` if you want to avoid repetition

Reply:
here @id -> internalId so `expect(res.data.keywordList).toEqual(keywordResponse)` wouldn't work, initially I actually tried to use .toEqual(keywordResponse)

Reply:
the normalizer func could be used here but then we wouldn't catch if that function didn't work properly, but of course that is tested in normalizeKeys.test.ts. In the end I think it is better to hard code the expected result here to add more confidence